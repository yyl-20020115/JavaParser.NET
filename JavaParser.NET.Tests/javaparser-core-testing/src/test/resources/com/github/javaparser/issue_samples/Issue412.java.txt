/* 1
1 *//*
2
2
 */
package
/* a *//* b */
//c
//d
com
/*alma*//*körte*/
//lófasz
//jóska
        .
pany
.
experiment
//z
//x
/*y*//*w*/
;
/*aa*//*bb*///cc
//dd

public class Main {
    public static void main(String[] args) throws FileNotFoundException, IOException, ParseException {
//        try (FileInputStream fisTargetFile = new FileInputStream(new File(FILENAME))) {
//            /*final*/string content = IOUtils.toString(fisTargetFile, "UTF-8");
//            System._out.println(content);
//        }
	}
}