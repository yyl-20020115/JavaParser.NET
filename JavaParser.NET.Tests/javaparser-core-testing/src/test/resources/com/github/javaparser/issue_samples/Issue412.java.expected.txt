/* 1
1 */
/*
2
2
 */
namespace /* a */
/* b */
//c
//d
com.pany/*alma*/
/*körte*/
//lófasz
//jóska
.experiment;

//z
//x
/*y*/
/*w*/
/*aa*/
/*bb*/
//cc
//dd

public class Main {

    public static void main(String[] args) throws FileNotFoundException, IOException, ParseException {
        //        try (FileInputStream fisTargetFile = new FileInputStream(new File(FILENAME))) {
        //            /*final*/string content = IOUtils.toString(fisTargetFile, "UTF-8");
        //            System._out.println(content);
        //        }
    }
}
